docs/building-a-bento-amp-extension.md

```diff
@@ -92,130 +93,98 @@ The directory structure is below:
 
 In most cases you'll only create the required (req'd) files. If your element does not need custom CSS or JSS, you don't need to create those files.
 
-## Extend AMP.PreactBaseElement
-
-All Preact-based Bento AMP extensions extend `AMP.PreactBaseElement`. Bento AMP extensions differ from classic AMP extensions because they are self-managing and independent, and therefore usable in a wider range of contexts beyond AMP pages, while still being fully integrated with the AMP environment when used in a fully AMP document.
-
-Important: Bento-enabled AMP components built upon a Preact component must also be usable in insolation in React development contexts.
-
-The configurations which bridge the Preact implementation of the component and its custom element counterpart in an HTML or AMP document are explained in the [AMP/Preact Bridge](#amppreact-bridge) section, and the callbacks which handle AMP- and DOM- specific mutability traits are explained in the [PreactBaseElement Callbacks](#preactbaseelement-callbacks) section. All of these are also explained inline in the [PreactBaseElement](https://github.com/ampproject/amphtml/blob/main/src/preact/base-element.js) class.
-
-### Element and Component classes
-
-The following shows the overall structure of your element implementation
-file (`extensions/amp-my-element/1.0/amp-my-element.js`). See [Experiments](#experiments) to make sure your component is experimentally gated if necessary.
+### Versioning
 
-```js
-import {func1, func2} from '../../../src/module';
-import {BaseElement} from './base-element'; // Preact base element.
-import {CSS} from '../../../build/amp-my-element-1.0.css';
-// more ES2015-style import statements.
+AMP runtime is currently in v0 major version. Extensions versions are
+maintained separately. If your changes to your non-experimental
+extension makes breaking changes that are not backward compatible you
+should version your extension. This would usually be by creating a 0.2
+directory next to your 0.1.
 
-/** @const */
-const EXPERIMENT = 'amp-my-element';
+When version 0.2 is under development, make sure that `latestVersion` is
+set to 0.1 for both the 0.1 and 0.2 entries in `extensionBundles`. Once 0.2
+is ready to be released, `latestVersion` can be changed to 0.2.
 
-/** @const */
-const TAG = 'amp-my-element';
+If your extension is still in experiments breaking changes usually are
+fine so you can just update the same version.
 
-class AmpMyElement extends BaseElement {
-  /** @override */
-  init() {
-    // Perform any additional processing of prop values that are not
-    // straightforward attribute passthroughs, as well as AMP-specific
-    // work such as registering actions and events.
-    this.registerApiAction('close', (api) => api.close());
+Note that Bento upgrades to existing AMP components should go by one major version, meaning it should create a 1.0 directory next to an existing 0.1 or 0.2.
```

Comment:
nit: To maintain consistency with the rest of the document, please format sentences to a single line instead of manually wrapping them.